packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js

```diff
@@ -40,6 +41,14 @@ export default class TrafficController {
       this.model = this.selectedIp;
       this.getTraffic();
     }
+    this.ip = this.$stateParams.ip;
+    this.dateTime = this.$stateParams.dateTime
```

Comment:
```suggestion
```
No need because this.ip and this.dateTime is already defined by routing an binded in component so you can directly call this.ip and this.dateTime without get in $stateParams again.